infra-as-code/bicep/main.bicep

```diff
@@ -47,7 +47,7 @@ module keyVaultModule 'keyvault.bicep' = {
 
 // Deploy Azure Container Registry
 module acrModule 'acr.bicep' = {
-  name: 'acrDeploy'
+  name: 'acrDeploy-${uniqueString(deployment().name)}'

```

Comment:
Why is this the only module that has this suffix?

Reply:
I ran into deployment errors due to other workflows running other deployments on the same rg, not required anymore, removed.